skrf/network.py

```diff
@@ -5663,23 +5682,24 @@ def s2a(s: npy.ndarray, z0: NumberLike = 50) -> npy.ndarray:
         raise IndexError('abcd parameters are defined for 2-ports networks only')
 
     z0 = fix_z0_shape(z0, nfreqs, nports)
-    z01 = z0[:,0]
-    z02 = z0[:,1]
-    denom = (2*s[:,1,0]*npy.sqrt(z01.real * z02.real))
+    z01 = z0[:, 0]
+    z02 = z0[:, 1]
+    denom = (2 * s[:, 1, 0] * npy.sqrt(z01.real * z02.real))
     a = npy.array([
         [
-            ((z01.conj() + s[:,0,0]*z01)*(1 - s[:,1,1]) + s[:,0,1]*s[:,1,0]*z01) / denom,
-            ((1 - s[:,0,0])*(1 - s[:,1,1]) - s[:,0,1]*s[:,1,0]) / denom,
+            ((z01.conj() + s[:, 0, 0] * z01) * (1 - s[:, 1, 1]) + s[:, 0, 1] * s[:, 1, 0] * z01) / denom,
+            ((1 - s[:, 0, 0]) * (1 - s[:, 1, 1]) - s[:, 0, 1] * s[:, 1, 0]) / denom,
```

Comment:
While I understand the change from  `s[:,m,n]` to `s[:, m, n]`, I'm not sure if this helps to read in such cases...

Reply:
I'd say it's fine to change, as the latter is pep8 conform. (E231) 
https://www.flake8rules.com/rules/E231.html